abci/tests/test_cli/test.sh

```diff
@@ -38,8 +38,34 @@ function testExample() {
 	rm "${INPUT}".out.new
 }
 
+function testHelp() {
+	INPUT=$1
+	APP="$2 $3"
+
+	echo "Test: $APP"
+	$APP &> "${INPUT}.new" &
+	sleep 2
+
+	pre=$(shasum < "${INPUT}")
+	post=$(shasum < "${INPUT}.new")
+
+	if [[ "$pre" != "$post" ]]; then
+		echo "You broke the tutorial"
+		echo "Got:"
+		cat "${INPUT}.new"
+		echo "Expected:"
+		cat "${INPUT}"
+		echo "Diff:"
+		diff "${INPUT}" "${INPUT}.new"
+		exit 1
+	fi
+
+	rm "${INPUT}".new
+}
+
 testExample 1 tests/test_cli/ex1.abci abci-cli kvstore
 testExample 2 tests/test_cli/ex2.abci abci-cli kvstore
+testHelp tests/test_cli/testHelp.out abci-cli help
```

Comment:
The broken command is `abci-cli help kvstore`.
To make sure, I've applied your patch, removed the change in `abci/cmd/abci-cli/abci-cli.go`, but left those in `test.sh` and the tests still pass.

Reply:
in my local `abci-cli help` was also broken (basically, every help with a command is broken) before the patch, it will launch the abci_server. I will verify the test case without the patch again. Thanks for checking.

Reply:
NP, thanks for the help

Reply:
You're right. I re-ran the same I did a few days ago (removing the fix, re-running the test, expecting it to fail) and it failed indeed.
I must have made a mistake when I did it the first time.
Sorry for the noise